data/campaigns/fri04.wmf/scripting/starting_conditions.lua

```diff
@@ -0,0 +1,292 @@
+   smoked_fish = 5,
+   bread_frisians = 5,
+   honey_bread = 5,
+   beer = 5,
+   mead = 5,
+   fruit = 10,
+   honey = 5,
+}
+hq:set_workers {
+   frisians_carrier = 40,
+   frisians_builder = 5,
+   frisians_blacksmith = 3,
+   frisians_baker = 1,
+   frisians_brewer = 1,
+   frisians_claydigger = 2,
+   frisians_brickmaker = 2,
+   frisians_reed_farmer = 2,
+   frisians_berry_farmer = 2,
+   frisians_fruit_collector = 2,
+   frisians_farmer = 2,
+   frisians_landlady = 2,
+   frisians_smoker = 2,
+   frisians_geologist = 2,
+   frisians_woodcutter = 3,
+   frisians_forester = 2,
+   frisians_miner = 4,
+   frisians_smelter = 1,
+   frisians_stonemason = 2,
+}
+
+local total_soldiers = 0
+local soldiers = {50, 25, 5}
+for descr,n in pairs(campaign_data.soldiers) do total_soldiers = total_soldiers + n end
+if total_soldiers < soldiers[difficulty] then
+   if campaign_data.soldiers["000"] then
+      campaign_data.soldiers["000"] = campaign_data.soldiers["000"] + soldiers[difficulty] - total_soldiers
+   else
+      campaign_data.soldiers["000"] = soldiers[difficulty] - total_soldiers
+   end
+end
+soldiers = {}
+for descr,n in pairs(campaign_data.soldiers) do
+   soldiers[{tonumber(descr:sub(1,1)), tonumber(descr:sub(2,2)), tonumber(descr:sub(3,3)), 0}] = n
+end
+hq:set_soldiers(soldiers)
+
+-- =======================================================================
+--                                 Player 2
+-- =======================================================================
+p2.hidden_from_general_statistics = true
+p2:allow_buildings("all")
+p2:forbid_buildings {
+   "frisians_port",
+   "frisians_shipyard",
+   "frisians_weaving_mill",
+}
+
+hq = p2:place_building("frisians_headquarters", map.player_slots[2].starting_field, false, true)
+hq:set_wares {
+   brick = 50,
+   log = 50,
+   reed = 50,
+   granite = 50,
+   coal = 30,
+   iron = 20,
+   iron_ore = 20,
+   gold = 10,
+   gold_ore = 10,
+   ration = 30,
+   meal = 10,
+   bread_frisians = 15,
+   honey_bread = 5,
+   beer = 15,
+   mead = 5,
+   fruit = 30,
+   honey = 10,
+   water = 20,
+   barley = 30,
+   clay = 20,
+   fish = 50,
+   meat = 5,
+   smoked_meat = 60,
+   smoked_fish = 20,
+}
+hq:set_workers {
+   frisians_carrier = 40,
+   frisians_builder = 5,
+   frisians_blacksmith = 3,
+   frisians_baker = 1,
+   frisians_brewer = 1,
+   frisians_claydigger = 2,
+   frisians_brickmaker = 2,
+   frisians_reed_farmer = 2,
+   frisians_berry_farmer = 2,
+   frisians_fruit_collector = 2,
+   frisians_farmer = 2,
+   frisians_landlady = 2,
+   frisians_smoker = 2,
+   frisians_geologist = 2,
+   frisians_woodcutter = 3,
+   frisians_forester = 2,
+   frisians_miner = 4,
+   frisians_stonemason = 2,
+}
+soldiers = {
+   {{2,6,2,0}, 30},
+   {{1,3,1,0}, 20},
+   {{0,0,0,0}, 10},
+}
+hq:set_soldiers(soldiers[difficulty][1], soldiers[difficulty][2])
+
+p2:place_building("frisians_tower", map:get_field(13,13), false, true):set_soldiers(soldiers[difficulty][1], 7)
```

Comment:
if these buildings are meant as a safeguard for Reebaud they should be made undestructible. Else the Ai will dismantle them.